### ISSUE TITLE
Add Grids for Brazil

### ISSUE THREAD

**Issue by luigicc** [NaturalGIS/ntv2_transformations#18]
Hello!

The Brazilian grids can be download (bundled with software ProGrid) here: ftp://geoftp.ibge.gov.br/aplicativos/transformacao_coordenadas/

There is some information about that in: https://wiki.osgeo.org/wiki/Brazilian_Coordinate_Reference_Systems

I attached the files for convenience.

SAD96_003.GSB: EPSG:5527 -> EPSG:4674 (Transformation EPSG:5529)
SAD69_003.GSB: EPSG:4618 -> EPSG:4674 (Transformation EPSG:5528)
CA61_003.GSB:     EPSG:5524 -> EPSG:4674 (Transformation EPSG:5525)
CA7072_003.GSB: EPSG:4225 -> EPSG:4674 (Transformation EPSG: 5526)

[GridFiles.zip](https://github.com/NaturalGIS/ntv2_transformations/files/240551/GridFiles.zip)

I will look for some info about the grid file license.

Thanks!

**Comment by gioman** [comment]
Thanks. Without knowing for sure that the grids can be redistributed then I will not add them to the plugin. Anyway, even is it will be ok to redistribute them I would need some support to add them to the plugin, or you can propose a pull request with the necessary changes.

**Comment by luigicc** [comment]
Hello,

i receive a respond via email from IBGE - the national institute responsible by ProGrid:
"Informamos que dados selecionados de publicações do IBGE, apresentados em textos, bases de dados, tabelas, gráficos, mapas e imagens, que tenham como fonte exclusiva o IBGE, podem ser utilizados devendo ser citado como fonte o IBGE. Vale ressaltar o item 1.6 da R.PR nº 01/2015 (veja link abaixo) que diz que o IBGE não se responsabiliza pelo uso das grades fora do ProGriD."

tl;dr : We can use the grids. Condition: Cite the IBGE as source.
I send this email via the website http://www.ibge.gov.br/home/disseminacao/online/webmaster/default.shtm, "Dúvidas sobre as informações divulgadas no site". This email generate a protocol number, so i understand this as an oficial statement. Should i forward the email to you?

**Comment by luigicc** [comment]
Hello @gioman !

There is a pdf with some information relevant to this grids. 
[rpr_01_2015_sirgas2000.pdf](https://github.com/NaturalGIS/ntv2_transformations/files/246851/rpr_01_2015_sirgas2000.pdf)

**Comment by gioman** [comment]
@luigicc Olá! thanks for the informations, I will add the transformations for Brazil (it would be very good if you can attach here the necessary sample data to test them) as soon as possible, that could not be very soon unless someone is willingly to support the necessary work.

**Comment by aguiarbsb** [comment]
Hello gioman

Any news about inserting the transformations of Brazil?

**Comment by gioman** [comment]
@aguiarbsb 

> Hello gioman
> 
> Any news about inserting the transformations of Brazil?

Hi, it would take a few hours of work, and at this time I'm not able to do that in my free time. If you need feel free to submit a patch or to hire me. Cheers!
